### DISCUSSION TITLE
Welcome to lottie-flutter Discussions!

### DISCUSSION THREAD

**Discussion by xvrh** [xvrh/lottie-flutter#115] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by francyfre** [comment]
Hello to everybody...
Someone know why Lottie.asset('assets/...') make this error in ios deploy:
Error (Xcode): ../../../.pub-cache/hosted/pub.dev/lottie-2.2.0/lib/src/providers/asset_provider.dart:32:18: Error: Expected ';' after this.
Everything is ok in pubspect.yaml so i dont understand whats problem there is.

ThankYouuu

francyfre

**Reply by xvrh** [reply]
Weird. Can you try `flutter pub cache repair`?
Also which version of flutter are you using?

**Reply by francyfre** [reply]
ThankYouVeryMuch... is all solved by your suggestion <3
